mkt/webapps/indexers.py

```diff
@@ -423,11 +424,17 @@ def run_indexing(cls, ids, ES=None, index=None, **kw):
 
         qs = Webapp.with_deleted.no_cache().filter(id__in=ids)
         ES = ES or cls.get_es()
-
         docs = []
-        for obj in list(qs):
```

Comment:
calling `list()` was a guess at not leaving a queryset open when the limit's hit, but obviously more db access is done in `extract_document` etc.